### PR TITLE
feature: added context-path support

### PR DESCRIPTION
**Pull request by xamien**
I follow up on pull request for route prefix https://github.com/zeebe-io/zeebe-simple-monitor/pull/171/ using
`server.servlet.context-path`

@shawnsarwar

### CODE REVIEW

File: src/main/resources/public/js/app.js [hunk 1]
```diff
@@ -53,7 +53,7 @@ function reload() {
             }
              
             function sendMessage(msg) {
-                stompClient.send("/notifications", {},
+                stompClient.send(base_path + "notifications", {},
```
Comment (saig0) [inline src/main/resources/public/js/app.js:56]: Following DRY ("dont-repeat-yourself"), we could extract the building of the path into a method.

File: src/main/java/io/zeebe/monitor/rest/ViewController.java [hunk 2]
```diff
@@ -99,6 +105,7 @@ public String workflowList(Map<String, Object> model, Pageable pageable) {
 
     model.put("workflows", workflows);
     model.put("count", count);
+    model.put("context-path", base_path);
```
Comment (saig0) [inline src/main/java/io/zeebe/monitor/rest/ViewController.java:108]: Following DRY, we can extract this part into a method.

**Review by saig0 — CHANGES_REQUESTED** [review]
@xamien thank you for your contribution :tada: 

It looks good :+1: I just added some minor suggestions.
Please have a look. 

And please have a look at the CLA bot. Maybe, the email of the commit is different from GitHub. See:

> Michal Kotek seems not to be a GitHub user. You need a GitHub account to be able to sign the CLA. If you have already a GitHub account, please add the email address used for this commit to your account.

**Comment by xamien** [comment]
@saig0 thank you for review. I hope that I worked all comments.

**Review by saig0 — APPROVED** [review]
Looks good :tada: 

I'll add some minor changes myself to clean up.

**Pull request merged**